app/src/main/res/layout/profile_list_profile_view.xml

```diff
@@ -25,6 +25,7 @@
       android:layout_marginTop="18dp"
       android:layout_marginBottom="18dp"
       android:background="@drawable/profile_image_shadow"
+      android:elevation="10dp"
```

Comment:
Why this difference compared to the other views? Also, I'm wondering if this should be part of the common style in the same way the shape form is.

Reply:
> Why this difference compared to the other views? Also, I'm wondering if this should be part of the common style in the same way the shape form is.

The difference was due to its size as it was smaller compared to the other ones.Although now I changed all of them to elevation of 8dp.
I tried to add this in common style but it is again resulting in invisible shadow.

Reply:
Can you show what the new before & afters look like (the version without a shadow isn't needed, I think we're just interested now in the difference between the shadows) using the 8dp elevation?

Regarding the common style, can you provide a Gist with the styles change to provide context on what you tried? It'd be good to understand that. I'm not sure if this is a platform limitation, but gathering more data can be helpful.

Reply:
> Why this difference compared to the other views? Also, I'm wondering if this should be part of the common style in the same way the shape form is.

I guess I misunderstood here something.
Can you please elaborate what exactly I m supposed to do regarding this common style?

Reply:
> Can you show what the new before & afters look like (the version without a shadow isn't needed, I think we're just interested now in the difference between the shadows) using the 8dp elevation?
> 
> Regarding the common style, can you provide a Gist with the styles change to provide context on what you tried? It'd be good to understand that. I'm not sure if this is a platform limitation, but gathering more data can be helpful.


| Before (for 20dp elevation)  | After (for 8dp elevation) |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/156288347-d2299bd1-8db9-419d-8177-bfc8aaf67551.jpeg" width="250" />|<img src="https://user-images.githubusercontent.com/78796264/156288211-26ee6f9e-33b1-4385-aa63-e69835d03a3b.jpeg" width="250" />|

Reply:
@bhaktideshmukh See https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists for what a "Gist" is.

Reply:
Sorry to clarify @bhaktideshmukh I'm interested in seeing the comparison of the current CircularImageView vs. ShapeableImageView with 8dp (since that's the main consideration). However, with the images you provided I can do a side-by-side with those two.

I think the 8dp shadow still looks a bit thicker than the previous version. Can you confirm and maybe try something like 4dp, instead? This is actually what I meant by 'experimentally finding the correct shadow': you should change the shadows and compare with the original images produced by CircularImageView until the shadow looks nearly the same.

Reply:
> Sorry to clarify @bhaktideshmukh I'm interested in seeing the comparison of the current CircularImageView vs. ShapeableImageView with 8dp (since that's the main consideration). However, with the images you provided I can do a side-by-side with those two.
> 
> I think the 8dp shadow still looks a bit thicker than the previous version. Can you confirm and maybe try something like 4dp, instead? This is actually what I meant by 'experimentally finding the correct shadow': you should change the shadows and compare with the original images produced by CircularImageView until the shadow looks nearly the same.

Can you please have a look at the screenshots:--
I have added screenshots for different elevations -- 5dp,4dp,3dp.
I think 4 or 5 dp is giving closer view to the original CircularImageView.
Once you approve the final elevation I'll push the changes here.

**Elevation of 5dp**

| CircularImageView  | ShapeableImageView |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/153030088-1225bb3f-74d0-4a31-8801-3560006660aa.jpeg" width="280" />|<img src="https://user-images.githubusercontent.com/78796264/156711505-84a4fbcb-7c75-4f4b-bebd-19f0c2cef50f.jpg" width="280" />|

**Elevation of 4dp**

| CircularImageView  | ShapeableImageView |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/153030088-1225bb3f-74d0-4a31-8801-3560006660aa.jpeg" width="280" />|<img src="https://user-images.githubusercontent.com/78796264/156711718-39c3c0e5-8a7e-4aad-85a3-b42b76035c24.jpg" width="280" />|

**Elevation of 3dp**

| CircularImageView  | ShapeableImageView |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/153030088-1225bb3f-74d0-4a31-8801-3560006660aa.jpeg" width="280" />|<img src="https://user-images.githubusercontent.com/78796264/156711726-792bf184-a33e-4f2f-b3b0-f51086e8fce5.jpg" width="280" />|